src/Simple.OData.Client.Core/Fluent/CommandDetails.cs

```diff
@@ -14,6 +14,7 @@ class CommandDetails
         public string DynamicPropertiesContainerName { get; set; }
         public string FunctionName { get; set; }
         public string ActionName { get; set; }
+        public bool IsAlternateKey { get; set; }
```

Comment:
A comment pointing to the current community spec would be helpful for future editors.

Reply:
Please elaborate, I'm not sure what I have to do here :)

Reply:
I was unfamiliar with the alternate keys syntax. Basically adding a comment with a pointer to the spec that governs them would be helpful: https://github.com/OData/vocabularies/blob/master/OData.Community.Keys.V1.md

Reply:
Alright, want that comment right there on the IsAlternateKey property?

Reply:
After reviewing your updated changes I think this would be best in a the V4 Metadata code by the `GetAlternateKeyPropertyNames`.

Reply:
Added summary comments to interface declaration and both V3/V4 implementations of GetAlternateKeyPropertyNames. It would be nice if this were done for all the metadata methods.